src/status/mod.rs

```diff
@@ -77,6 +78,8 @@ pub trait StatusBackend {
             None,
         )
     }
+
+    fn dump_to_stderr(&mut self, output: &[u8]);
```

Comment:
If this is going to go inside the base trait, I think it should have a more generic name ... `dump_error_logs` or something? Also, perhaps the bit in the CLI that delimits the dump with equals signs could go inside this function? That feels like it might work nicely.